src/mysql_k8s_helpers.py

```diff
@@ -189,6 +213,39 @@ def configure_instance(self) -> None:
             )
             raise MySQLConfigureInstanceError(e.message)
 
+    def reconfigure_instance(self) -> None:
+        """Reconfigure the instance to be used in an InnoDB cluster.
+
+        Assumes that the instance was previously configured, and that the cluster
+        admin user already exists.
+        """
+        options = {
+            "restart": "false",
```

Comment:
Any reason why restart is false?
Ability to do restarts is one of the critical things needed for self-healing InnoDB Cluster.

Reply:
Restart is false because `mysql-shell` by default tries to restart a systemd service. However, we disable the restart by mysql-shell and use `pebble` to restart the `mysqld` service